src/components/layout/nav.js

```diff
@@ -266,7 +259,7 @@ const LinkSignupButton = styled(LinkButton)`
 const HamburgerMenu = styled.img`
     cursor: pointer;
     display: none;
-    @media ${device.tabletL} {
+    @media (max-width: 1060px) {
```

Comment:
```suggestion
    @media ${device.bp1060} {
```

Reply:
This will need to be added into `device.js`, but sounds good 👍